README.md

```diff
@@ -28,34 +28,105 @@ The ```winetricks``` package should be used if it is available and up to date. T
 
 Note: packaged Debian / Ubuntu winetricks versions are typically outdated, so a manual installation is recommended.
 
-If the package is unavailable, outdated, or the latest version is desired, a manual installation of winetricks can be done. E.g.:
+If the package is unavailable, outdated, or the latest version is desired, a manual installation of winetricks can be done.
+It is recommended to uninstall any previously installed version of winericks first.
 
+**_If you don't uninstall a previously installed, packaged version of winetricks... Well then you get to pick up the pieces!_**
+
+E.g. for Debian / Ubuntu:
+```
+sudo apt-get purge winetricks
 ```
-sudo -- sh -c '
-cat > /usr/local/bin/update_winetricks.sh <<_EOF_SCRIPT
+
+Then, for Ubuntu, use a simple shell script to download the current winetricks scripts.
+This script uses a single parameter (to set a working directory).
+E.g.:
+
+```
+# Change to a directory writeable by current user (their HOME directory). See:
+#   https://www.tldp.org/LDP/abs/html/internalvariables.html
+cd "${HOME}"
```

Comment:
If you want to cd to $HOME, you can just use 'cd'

Reply:
I was trying to make the cd destination directory explicit - for "new users".